gruntfile.js

```diff
@@ -1,112 +1,118 @@
 module.exports = function(grunt) {
-    // Project Configuration
-    grunt.initConfig({
-        pkg: grunt.file.readJSON('package.json'),
-        watch: {
-            jade: {
-                files: ['app/views/**'],
-                options: {
-                    livereload: true,
-                },
-            },
-            js: {
-                files: ['public/js/**', 'app/**/*.js'],
-                tasks: ['jshint'],
-                options: {
-                    livereload: true,
-                },
-            },
-            html: {
-                files: ['public/views/**'],
-                options: {
-                    livereload: true,
-                },
-            },
-            sass: {
-                files: ['public/css/common.scss, public/css/views/articles.scss'],
-                tasks: ['sass:dist']
-            },
-            css: {
-                files: ['public/css/**'],
-                tasks: ['sass'],
-                options: {
-                    livereload: true
-                }
-            }
+  // Project Configuration
+  grunt.initConfig({
+    karma: {
+      unit: {
+        configFile: "./test/karma.conf.js"
```

Comment:
Not sure if we should update this file considering that we have a chore to convert to using gulp as out task runner. Please sync with @kenoseni on this.